spec/models/bookmarks_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe Bookmark do
+  it 'allows user to create 5 bookmarks' do
+    user = user_with_many_bookmarks(5)
+    expect { user.save! }.not_to raise_error(ActiveRecord::RecordInvalid)
+    expect(user.bookmarks.count).to eq(5)
+  end
+  it 'does not allow user to create more than 1000 bookmarks' do
```

Comment:
Is it possible to override `MAXIMUM_BOOKMARKS` just for this test, so that we don't have to create so many objects in the database (slowing down the test suite)?